xbmc/utils/BitstreamConverter.cpp

```diff
@@ -726,6 +727,18 @@ bool CBitstreamConverter::BitstreamConvertInitAVC(void *in_extradata, int in_ext
       if (unit_nb)
         pps_seen = 1;
     }
+
+    if (!unit_nb && !mvc_done++)
```

Comment:
I know you were following the pattern but generally ints either need to be compared directly or casted to bool. I also recommend using a bool when wanting a boolean check.
```suggestion
    if (unit_nb == 0 && !mvc_done)
```